### PR TITLE
semgrep 0.28.0

### PR DESCRIPTION
**Pull request by brendongo**
Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

---

[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

### CODE REVIEW

File: Formula/semgrep.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class Semgrep < Formula
   desc "Easily detect and prevent bugs and anti-patterns in your codebase"
   homepage "https://semgrep.dev"
   url "https://github.com/returntocorp/semgrep.git",
-    tag:      "v0.27.0",
-    revision: "549bfa79f81939ac06e57a527b5f0d46c87c46ce"
+    tag:      "v0.28.0",
+    revision: "bc9375f8d42bbf89a715638894d58e87da974748"
```
Comment (underyx) [inline Formula/semgrep.rb:8]:
```suggestion
    revision: "e5670ed176e18a6927fea85d2a82212b5d272806"
```

Needs bump after https://github.com/returntocorp/semgrep/pull/1862/commits/e5670ed176e18a6927fea85d2a82212b5d272806

**Comment by underyx** [comment]
@brendongo I'll need you to accept this suggestion, I can't edit your branch and I'm not supposed to open duplicate PRs I believe.

File: Formula/semgrep.rb [hunk 2]
```diff
@@ -137,6 +137,14 @@ def install
 
       system "opam", "exec", "--", "make", "setup"
 
+      # Install spacegrep
+      cd "spacegrep" do
```
Comment (SMillerDev) [inline Formula/semgrep.rb:141]: Please add a test for this.

Comment (brendongo) [reply Formula/semgrep.rb:141]: When semgrep initializes it verifies that semgrep-core and spacegrep are on path. https://github.com/returntocorp/semgrep/blob/684302680fa07c6f7a884750c8efaad0df777587/semgrep/semgrep/util.py#L148 so existing tests will fail if this is not installed correctly

**Review by SMillerDev — APPROVED** [review]
Thanks @brendongo  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**